hack/e2e-suite/services.sh

```diff
@@ -34,6 +34,24 @@ source "${KUBE_VERSION_ROOT}/cluster/${KUBERNETES_PROVIDER}/util.sh"
 
 prepare-e2e
 
+echo "Creating test namespace ${NAMESPACE}"
+${KUBECTL} create -f - << __EOF__
+{
+    "kind": "Namespace",
+    "apiVersion": "v1beta3",
+    "metadata": {
+        "name": "${NAMESPACE}"
+    },
+    "spec": {
+        "finalizers": [
+            "kubernetes"
```

Comment:
I'm so confused - I can not reproduce the error?!?!

On Tue, Jun 2, 2015 at 5:17 PM, Brian Grant <notifications@github.com>
wrote:

> In hack/e2e-suite/services.sh
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/9148#discussion_r31584393>
> :
>
> > @@ -34,6 +34,24 @@ source "${KUBE_VERSION_ROOT}/cluster/${KUBERNETES_PROVIDER}/util.sh"
> >
> >  prepare-e2e
> >
> > +echo "Creating test namespace ${NAMESPACE}"
> > +${KUBECTL} create -f - << __EOF__
> > +{
> > +    "kind": "Namespace",
> > +    "apiVersion": "v1beta3",
> > +    "metadata": {
> > +        "name": "${NAMESPACE}"
> > +    },
> > +    "spec": {
> > +        "finalizers": [
> > +            "kubernetes"
>
> This PR fixes one obvious problem -- the non-existent and unnecessary
> namespace shouldn't be specified when creating the namespace. That
> generates an error, if nothing else.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/9148/files#r31584393>
> .
>